libraries/rib-base/src/main/java/com/badoo/ribs/routing/state/feature/PendingTransition.kt

```diff
@@ -17,7 +18,7 @@ internal class PendingTransition<C : Parcelable>(
     private val transitionElements: List<TransitionElement<C>>,
     private val effectEmitter: EffectEmitter<C>,
     private val internalTransactionConsumer: InternalTransactionConsumer<C>,
-    private val handler: Handler = Handler()
+    private val handler: Handler = Handler(Looper.getMainLooper())
```

Comment:
Please check this, do we really need Handler()? It can be created not on the main thread in that case